ecl/hql/hqlgram2.cpp

```diff
@@ -12272,11 +12272,12 @@ bool parseForwardModuleMember(HqlGramCtx & _parent, IHqlScope *scope, IHqlExpres
 void parseAttribute(IHqlScope * scope, IFileContents * contents, HqlLookupContext & ctx, IIdAtom * name, const char * fullName)
 {
     bool alreadySimplified = false;
+    bool excludeFromCache = ctx.excludeFromCache(fullName);
```

Comment:
This should actually be a neverSimplify option.  You should still generate a cache entry (otherwise quick archive generation will not work), but never simplify the expression.